### PR TITLE
[#15] Tools page

### PR DESCRIPTION
**Pull request by qlemaire22**
First draft of the tools page. https://github.com/stormgateworld/web/issues/15

I used the content Card for it but I had to make a bunch of field optional, is that OK? Should I follow another approach? 

Also, I didn't know which tools/projects you wanted to highlight so I put only one for now. I can put more if you have a list or anything.

### CODE REVIEW

**Comment by qlemaire22** [comment]
I can add tags like on aoeworld also. 
And I was wondering if the data should be hardcoded or if I should fetch it from some places?

**Comment by robertvanhoesel** [comment]
Have to take a detailed look later but hardcoded is fine!

You can add some more tools from the community like that Replay renamer, Luminite.gg etc

**Comment by qlemaire22** [comment]
I added tags + community tools.
Here is how community tools looks like for example.

Screenshot 2024-02-06 at 18 46 23

I left aligned the tags to be consistent with other Content metadata. 
I based the style of the tags to buttons + rounded borders. If you have anything else in mind I can improve it.

File: src/pages/tools.astro [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    url: "https://gist.github.com/acarapetis/b5e9cf199ea2a468ace2f756ac287268",
+    image_url: replayRenamerImage.src,
+    metadata: { tags: ["Replays", "External"] },
+  },
+]
+---
+
+<Layout title="Tools">
+  <div class="w-full bg-gray-800/50 backdrop-blur-lg border-b border-gray-700/50">
+    <div class="mx-auto max-w-screen-md px-4 md:px-0 py-8 gap-4 flex items-center flex-wrap justify-between">
```
Comment (robertvanhoesel) [inline src/pages/tools.astro:40]: I'm really happy with everything, except these headers.

I think you can best copy over the header from the Stats page and just replace the text.

If you want to split the tools into subheading you can copy the code that says 'Factions' as a header above the two groups of tools.

Alternatively you could just show 'offical' or 'by SGW' or something as a tag maybe. I'm fine either way

**Review by robertvanhoesel — CHANGES_REQUESTED** [review]
Looks great! Thanks, one minor thing about the headers though.

**Comment by qlemaire22** [comment]
I changed the header, here is how it looks like:

Screenshot 2024-02-07 at 08 54 02

I hope the wording is fine, I was not sure what to replace "Game" with. 
I also merged the two subdivisions as I think it is better for now since there are not so many projects. Could be revisited later.

**Review by robertvanhoesel — COMMENTED** [review]
Applied some small fixes to the alignment 👍

**Pull request merged**